tests/.#README.md

```diff
@@ -0,0 +1 @@
+CHRISTIANSA@WLNSBIO9060661.18864:1662983367
```

Comment:
This file should be added to gitignore, just in case there is any security issues with having your workstation id exposed on the public repo